.gitlab-ci.yml

```
@@ -290,6 +290,53 @@ docs:
     - chown -R nonroot:nonroot ./crate-docs
 
 
+codecov:
+  stage:                           workspace
+  <<:                              *docker-env
+  <<:                              *test-refs
+  needs:
+    - job:                         check-std
+      artifacts:                   false
+  variables:
+    # For codecov it's sufficient to run the fuzz tests only once.
+    QUICKCHECK_TESTS:              1
+    INK_COVERAGE_REPORTING:        "true"
+    CARGO_INCREMENTAL:             0
+    # Needed because `codecov` requires nightly features to work
+    # (see `-Z` in the `RUSTFLAGS` below).
+    RUSTC_BOOSTRAP:                1
```

```suggestion
    RUSTC_BOOSTRAP:                "1"
```
Hmm try this, I don't know why but it worked for me